### DISCUSSION TITLE
How is this program supposed to work?

### DISCUSSION THREAD

**Discussion by Bumblebee001** [ShankarBUS/ModernFlyouts#1213] [General]
I installed this program. Nothing seems to have changed. I tried double clicking and run as administrator. Nothing happens. What should I be doing that perhaps I am not?

**Comment by llluxuryman** [comment]
Hi, i have the same problem. Hope someone can help

**Reply by manstheram** [reply]
@llluxuryman It changes the _flyout_ (the popup) shown when you change your volume using the volume buttons. It also has flyouts for when you press brightness keys or toggle a lock key (e.g. caps lock, num lock, etc.).
The volume flyouts can also show the current song being played and stuff on those lines.
A settings menu is also available as an icon in the notification area in the taskbar.
**Does this answer your problem?**

**Reply by llluxuryman** [reply]
it doesnt show any icon in the task bar. P.S. I click setup, then trying to star tis app and nothing changes

**Reply by manstheram** [reply]
That must be a bug then. Try using one of the beta builds since they tend to run better.
